source/include/sigv4.h

```diff
@@ -162,7 +165,24 @@ typedef enum SigV4Status
      * Functions that may return this value:
      * - #SigV4_GenerateHTTPAuthorization
      */
-    SigV4MaxHeaderPairCountExceeded
+    SigV4MaxHeaderPairCountExceeded,
+
+    /**
+     * @brief The maximum number of query parameters was exceeded while parsing
+     * the query string input parameter.
```

Comment:
It will be helpful to customer if we mention which config to update for changing the value for `SigV4MaxQueryPairCountExceeded `

Reply:
Added